src/html/utils.ml

```diff
@@ -1,87 +1,3 @@
 (* Shared utility functions *)
 
 let optional_elt f ?a = function [] -> [] | l -> [ f ?a l ]
```

Comment:
I think we can move this somewhere else now :-)

Reply:
Done in [f868577](https://github.com/ocaml/odoc/pull/1251/commits/f868577f5f5f98383300fc98b578acf7495051f7)